docs/FAQ-benchpark-repo.rst

```diff
@@ -0,0 +1,27 @@
+.. Copyright 2023 Lawrence Livermore National Security, LLC and other
+   Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+
+   SPDX-License-Identifier: Apache-2.0
+
+=================================
+Benchmark Specs in benchpark/repo
```

Comment:
IMO any subsection title within an FAQ section should be a question, or it should be phrased like "how to do X". As phrased, I don't think it's clear to users what they could achieve by reading through these `FAQ-*` sections.